### ISSUE TITLE
Tamed dogs that are tied down will still follow you down stairs

### ISSUE THREAD

**Issue by Nameless-Survivor** [CleverRaven/Cataclysm-DDA#27014]
Tamed dogs that are tied down still manage to somehow teleport up/down the stairs. They aren't coming untied, they just teleport to the bottom of the stairs and become tied down there. If you go back up, they follow you, and then become tied down on top of the stairs upstairs.

**To Reproduce**  
1. Tie a dog down with a staircase somewhere in the map tile.
2. Go downstairs.
3. Walk several tiles

**Expected behavior**  
Dogs should stay tied where they are?

**Versions and configuration(please complete the following information):**
 - OS: Windows 8.164 bit
 - Game Version: 0.C-34885-g3cb3f9e

**Comment by Ker-Nes** [comment]
Also have had it happen. In a potentially related issue, at least in my game, going up and down the stairs with the dog spawned a wolf after a bit, although it might have been just a random roll.

**Comment by Nameless-Survivor** [comment]
No, that happened to me too. I'd been wondering about that. 

I had a wolf spawn in the room I locked the dogs in for a short time. I needed their carriers to retrieve a bunch of chickens I found. I came back and found a wolf in the same room as the dogs. One of my NPCs was in there as well, they were bleeding, had a broken arm, leg, and 7 health left on the head. 

I was thinking it might be related to the predator/vermin spawn mechanics, because the three dogs I let out immediately shit all over the floor a LOT when I took them out of their carriers. 

Haven't had a chance to try and replicate it though, because once I found out the damn things can't be tied down effectively they went straight back into their containers.
